.travis.yml

```diff
@@ -4,44 +4,53 @@
 language: generic
 
 os: osx
-osx_image: beta-xcode6.1
+osx_image: xcode6.4
 
 env:
   matrix:
     
     - CONDA_PERL=5.20.3.1
+    - CONDA_PERL=5.22.0.1
+    - CONDA_PERL=5.22.2.1
```

Comment:
We should pin to PERL `5.20.3.1` (or latest?) to avoid this. There is no need for a PERL matrix and 3 versions of this package. (Not your fault. It is conda-smithy finding the PERL versions and trying to be smart.)